codebuild_specs/e2e_workflow_base.yml

```diff
@@ -7,7 +7,6 @@ env:
     AWS_REGION: us-east-1
     CDK_DEFAULT_REGION: us-east-1
     CLI_REGION: us-east-1
-    TEST_SUITE: src/__tests__/auth_2b.test.ts
```

Comment:
Changes in this file are to find some space as code build started to throw:
```
[Container] 2024/01/02 23:22:43.265930 Phase complete: DOWNLOAD_SOURCE State: FAILED
--
104 | [Container] 2024/01/02 23:22:43.265948 Phase context status code: YAML_FILE_ERROR Message: batch yaml definition exceeds size limit
```

Changes made:
- removed global line with env var (this is not used as every job defines it).
- removed compute type medium from jobs as global defaults define it above.